### ISSUE TITLE
SkBee says "SkullOwner" is an invalid block nbt [BUG]

### ISSUE THREAD

**Issue by superstorm192** [ShaneBeee/SkBee#314]
**Describe the bug**   
SkBee throws an exception saying that SkullOwner is an invalid nbt tag for a block. This used to be the case, but in 1.19.3 Mojang changed it to SkullOwner. Therefore `set tag "Owner" of {_comp} to tag "SkullOwner" of {_hcomp}` no longer works in Minecraft and `set tag "SkullOwner" of {_comp} to tag "SkullOwner" of {_hcomp}` throws an exception in SkBee

**To Reproduce**   
1. Open Minecraft and run a server in version 1.19.3
2. Create this skript for the server and enable it
```
command /skbeetest:
	permission: op
	usage: /skbeetest
	executable by: players
	trigger:
		set {_testitem} to player head with nbt compound from "{SkullOwner:{Id:[I;1642715083,-1150989963,-1404325767,285862586],Name:""Amethyst Chime"",Properties:{textures:[{Value:""eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvOGI1ZTNkZWNlNDI1YzhiNGNkNDY5MWY3NmEyMThjOGEwZDA0ODBiYTgwMmU2MTI4MmQwMmQwZDU2N2VkOGMzZSJ9fX0="", Signature:""Amethyst Chime""}]}},BlockEntityTag:{note_block_sound:""minecraft:block.amethyst_block.step""}}"
		give {_testitem} to player
		set block at player to player head
		set {_bcomp} to nbt compound of block at player
		set {_icomp} to nbt compound of {_testitem}
		#Note block sound works just fine
		set tag "note_block_sound" of {_bcomp} to tag "BlockEntityTag;note_block_sound" of {_icomp}
		#This does nothing
		set tag "Owner" of {_bcomp} to tag "SkullOwner" of {_icomp}
		#Uncomment for SkBee error
		#set tag "SkullOwner" of {_bcomp} to tag "SkullOwner" of {_icomp}
```
3. run the command /skbeetest and a default player head will appear at your feet
4. stand/stay on the player head
5. use `/data get block ~ ~ ~` to confirm the note block sound is there and the Owner is not.
6. place down the Amethyst Chime you received when using the command
7. use `/data get block ~ ~ ~` on the Amethyst Chime to confirm that the block data tag is indeed "SkullOwner" and no longer just "Owner"

**Expected behavior**   
`set tag "SkullOwner" of {_bcomp} to tag "SkullOwner" of {_icomp}` should set the player head the custom player head defined in the script

**Screenshots/Errors**   
https://hastebin.com/ijizixuruk.md

**Version Info:**
```
--- [SkBee Loading Info] ---
- [NBTAPI] Found Spigot: v1_19_R2! Trying to find NMS support
- [NBTAPI] NMS support 'MC1_19_R2' loaded!
- NBT Elements successfully loaded
- Recipe Elements successfully loaded
- Scoreboard Elements successfully loaded
- Scoreboard Objective Elements successfully loaded
- Team Elements successfully loaded
- Bound Elements successfully loaded
- Text Component Elements successfully loaded
- Pathfinding Elements successfully loaded
- Structure Elements successfully loaded
- Other Elements successfully loaded
- Virtual Furnace Elements disabled via config
- World Creator Elements successfully loaded
- Game Event Elements successfully loaded
- BossBar Elements successfully loaded
- Statistic Elements successfully loaded
- Villager Elements successfully loaded
- Advancement Elements successfully loaded
- World Border Elements successfully loaded
- Particle Elements successfully loaded
- Minecraft Tag elements successfully loaded
- RayTrace elements successfully loaded
- Plugin is up to date!
--- [Server Info] ---
Server Version: git-Paper-346 (MC: 1.19.3)
Skript Version: 2.6.4
Skript Addons:
 - skript-reflect v2.3
SkBee Version: 2.7.0
SkBee Website: https://github.com/ShaneBeee/SkBee
```

**Additional context**

**Comment by ShaneBeee** [comment]
So I used your code, and could not reproduce the error.

That said, the error sent does not match your issue.

1) In your issue here, you state you're using SkBee 2.7.0, yet in the error it shows SkBee 2.6.3
2) The line in the error:
```
set tag "SkullOwner" of {_comp} (as com.shanebeestudios.skbee.api.NBT.NBTCompound) to tag "SkullOwner" of {_hcomp} (as com.shanebeestudios.skbee.api.NBT.NBTCompound)
```
where did `{_hcomp}` come from?
also in the error I see:
```
....NbtApiException: [?]Tried getting invalide compound 'SkullOwner' from '{id:"minecraft:skull"}'!
```
It appears you're trying to get the tag "SkullOwner" from a compound that does not contain that tag.

That said:
1) I'm not saying your issue doesn't exist, I'm just saying I can not replicate it, with the information given
2) It would be more helpful if the information you provided matched the error you also provided

**Comment by superstorm192** [comment]
>  In your issue here, you state you're using SkBee 2.7.0, yet in the error it shows SkBee 2.6.3

I was using 2.6.3 and upgraded to 2.7.0 and forgot to update the error. The error no longer appears for "SkullOwner" yet it's still not working for the player head, it's still default steve instead of the amethyst chime.
> where did {_hcomp} come from?

it's supposed to be {_icomp}, somehow i pasted the wrong thing.
> I'm not saying your issue doesn't exist, I'm just saying I can not replicate it, with the information given

Well that's annoying, I don't know what information I am missing, it's pretty consice. Is the player head that gets put on the ground a steve head or an amethyst chime? Because for me it's still a steve head when using the line
`set tag "SkullOwner" of {_bcomp} to tag "SkullOwner" of {_icomp}` which is my main issue.

**Comment by ShaneBeee** [comment]
That doesn't work for me either. Never has.
I have NO CLUE why the heck it doesn't work. 
I don't think its an SkBee issue, it may be an issue with the API I am using for NBT (NBT-API)

What I have done in the past to make it work is set a compound to the skull data, and then ADD said compound to the block,
ex:
```vb
set {_n} to nbt compound from "{SkullOwner:...
add {_n} to nbt compound of block
```
this works, with one exception, the block visually does not update. Again, not sure why. The data is there, but it's almost like MC doesn't wanna send it to the client.

**Comment by ShaneBeee** [comment]
I fixed some issues with NBT in SkBee 2.7.1
These fixes should also fix your issue.
